py/desimeter/fieldmodel.py

```diff
@@ -121,14 +146,15 @@ def fit_tancorr(self,catalog,mjd=None,hexrot_deg=None,lst=None) :
         if mjd is not None :
             self.mjd = mjd
             log.info("Use argument MJD={}".format(self.mjd))
-        elif "mjd_obs" in catalog.keys():
-            self.mjd    = np.mean(catalog["mjd_obs"])
-            log.info("Use 'mjd_obs' in catalog, MJD={}".format(self.mjd))
-        elif self.mjd is None :
-            log.error("mjd is None")
-            raise RuntimeError("mjd is None")
-        else :
-            log.info("Use MJD={}".format(self.mjd))
+        if self.mjd is None :
+            if "mjd_obs" in catalog.keys():
+                self.mjd    = np.mean(catalog["mjd_obs"])
```

Comment:
I don't quite know  how the catalog is created, but I'd expect that the mjd_obs would be essentially the same for all objects (if we are processing one exposure), so I' don't quite get the mean(). If anything I'd assert if std(mjd_obs) is large enough